assets/scripts/script.js

```diff
@@ -50,6 +55,87 @@ function showRandomDrinkSection(){
 // SHOW RANDOM DRINK SECTION
 // button
 // id="randomDrink"
+// ***** FOR NARGIZA'S CODE FOR RANDOM DRINK start
+const getRandomDrinkBtn = document.getElementById('collins');
+const getRandomDrinkContainer = document.getElementById('randomDrinkResult');
+
+getRandomDrinkBtn.addEventListener('click', () => {
+	fetch('https://www.thecocktaildb.com/api/json/v1/1/random.php')
+		.then(res => res.json())
```

Comment:
I believe this can be done in the same line.  `.then(res => createRandomDrink(res.json().drinks[0]))`